CMR/python/cmr/util/common.py

```diff
@@ -80,3 +89,22 @@ def execute_command(cmd):
     """
     result = subprocess.run(cmd, stdout=subprocess.PIPE, stderr=subprocess.PIPE, check=True)
     return result.stdout.decode('utf-8')
+
+def call_security(account, service, app="/usr/bin/security"):
+    """Call the security command to look up encrypted values"""
```

Comment:
spacing in the doc string by the """ is off compared to others.

Reply:
I'm actually in the rules for once: "For one liner docstrings, please keep the closing """ on the same line" -> https://www.python.org/dev/peps/pep-0008/#documentation-strings